hitl/streaming.py

```diff
@@ -27,18 +28,36 @@ async def _main():
                         help="Maximum loss for success")
     args = parser.parse_args()
 
+    prefix = args.prefix
+    if not args.prefix:
+        devices = await miniconf.discover(args.broker, 'dt/sinara/dual-iir/+', 1)
```

Comment:
Why that?
Let's align this with the  miniconf py client: require a prefix, if it contains + then discover.

Reply:
This aligns with `hitl/loopback.py`. Specifically, it's to allow the operator to run the streaming HITL without having to provide a prefix at all.

I added this because I always forget the prefix and have to look through code, but it's super easy for me to just run `hitl/loopback.py --broker <my broker>`

Reply:
Ah. Mixed up the files. All good!